engine/src/main/java/org/terasology/rendering/nui/animation/SwipeMenuAnimationSystem.java

```diff
@@ -56,6 +56,8 @@ public float getVertScale() {
 
     private float scale;
 
+    private static float PAUSED_ANIMATION_FRAME_TICK_TIME = 0.035f;
```

Comment:
It took inspecting more closely in source while testing but this line remains slightly off:

1. It should be at the top of the class declaration (a check for things being in a standard order)
1. It should be declared as `private static final float ...` to be a true constant, without which it triggers a naming convention warning

I'm just committing that during the merge since I moved it to the top of the class and added `final` already trying to make the IDE happy. Had me scratching my head for a moment why it was unhappy about the name, but it wasn't the name as much as the missing `final` :-)

Not a big deal at all, works fine just project best practices and all that!

I also added the following TODO and will write up a separate issue with details:

`// TODO: Convert away from a faked time interval to handling UI updates to non-game objects in a non-gametime loop?`